smithy-linters/src/main/java/software/amazon/smithy/linters/NoninclusiveTermsValidator.java

```diff
@@ -62,44 +62,40 @@ public Provider() {
      * NoninclusiveTermsValidator configuration.
      */
     public static final class Config {
-        private Map<String, List<String>> appendNoninclusiveTerms = MapUtils.of();
-        private Map<String, List<String>> replaceNoninclusiveTerms = MapUtils.of();
+        private Map<String, List<String>> terms = MapUtils.of();
+        private Boolean excludeDefaults = Boolean.FALSE;
```

Comment:
This can just be a primitive `boolean`, since the null state isn't relevant.

Reply:
Fixed